src/test/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImplTest.java

```diff
@@ -677,6 +688,10 @@ private void assertCuDisbursementVoucherDocumentsCorrect(CuDisbursementVoucherDo
                 actualDvDocument.getDvNonEmployeeTravel().getDvNonEmployeeExpenses(), this::assertTravelExpenseCorrect);
         assertObjectListIsCorrect("non employee prepaid travel expenses aren't correct", expectedDvDocument.getDvNonEmployeeTravel().getDvPrePaidEmployeeExpenses(), 
                 actualDvDocument.getDvNonEmployeeTravel().getDvPrePaidEmployeeExpenses(), this::assertTravelExpenseCorrect);
+        assertEquals("Due Dates should match", dateFormat.format(actualDvDocument.getDisbursementVoucherDueDate()), 
+                dateFormat.format(expectedDvDocument.getDisbursementVoucherDueDate()));
+        assertEquals("Invoice Dates should match", actualDvDocument.getInvoiceDate(), expectedDvDocument.getInvoiceDate());
+        assertEquals("Invoice numbers should match", actualDvDocument.getInvoiceDate(), expectedDvDocument.getInvoiceDate());

```

Comment:
Is it safe to assume that the disbursementVoucherDueDate is non-null when it's passed to the date formatter? I think that's the case based on the default-due-date feature, but wanted to make sure.

Also, when calling assertEquals(), the arg for the expected value should precede the one for the actual value.

Reply:
Yes, the due should never be null, I shall switch the order